pkg/common/messages.go

```diff
@@ -11,16 +11,6 @@ type Messages struct {
 	Errors   []error
```

Comment:
change name of this pkg to resemble path pkg/messages/messages.go as it is not a good practice to have ambiguous names.